docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -260,7 +260,6 @@ spec:
     - configure-os
     - ssh-known-hosts
     - run-os
-    - reboot-os
```

Comment:
We need to note somewhere that this is just a WA. The solution should also prevent any reconfiguration by edpm_ansible on the EDPM node that would require the reboot-os role to be run. If we don't catch those reconfigurations the we create a timebomb on the EDPM node that will activate at the next unrelated reboot of the node.

Reply:
I think we need to file a bug describing the case when such reconfiguration happens during adoption (e.g. different kernelargs).

Reply:
I wonder if the "reboot os" role should be made somehow safe by default? If this role is in the list, and some of the sensitive parameters change, would it just reboot the whole dataplane? (Regardless if it was adopted or not...) Maybe the role should have some safety param that needs to be set for the reboots to be enabled, and if it detects it should reboot but this parameter isn't set, it should fail...

Reply:
Would that failure happen too late after some changes(e.g. another dataplane service run to completion) on the node that prevents the rollback to the old software version?